src/components/Layout/Nav/NavItem/index.tsx

```diff
@@ -23,13 +23,16 @@ export const NavItem = ({
   if (hidden) return null;
   const isExternal = isExternalLink(link);
 
+  if (isLottieOptions(prefix) || isLottieOptions(title)) return <></>;
```

Comment:
Why this change? This seems to apply to the Spacewalk button. Wouldn't that then skip rendering for this button?